docs/scripting/script-net.md

```diff
@@ -11,14 +11,14 @@ mentions:
 ---
 
 ::: warning
-The Script API is currently in active development, and breaking changes are frequent. This page assumes the format of Minecraft 1.19.80
+The Script API is currently in active development, and breaking changes are frequent. This page assumes the format of Minecraft 1.21.20
 :::
 
 ::: warning
-This module can only be used on Bedrock Dedicated Server.
+This module can only be used on Bedrock Dedicated Servers.
```

Comment:
Native module `@minecraft/server-net` can also be used in Editor extensions.

Reply:
@QuazChick Hypothetically, could you make an Editor addon that uploads a file to a server?